src/main/java/com/google/api/codegen/transformer/java/JavaSurfaceNamer.java

```diff
@@ -413,9 +413,9 @@ public String getAndSaveTypeName(ImportTypeTable typeTable, TypeModel type) {
    * <p>All Google cloud java libraries have package names like "com.google.cloud.library.v1" and
    * the respective examples have package names like
    * "com.google.cloud.examples.library.v1.snippets". This method assumes {@code packageName} has
-   * the format of "com.google.(.+).artifact.name". This works for both existing libraries and the
-   * baseline test. We will need to adjust this if in the future there are libraries that do not
-   * follow the package name format assumed here.
+   * the format of "com.google.orgname.artifact.name", where "artifact.name" could be empty. This
```

Comment:
This description doesn't seem quite right for the longrunning case - `longrunning` isn't an orgname.